src/hooks/useStoreDiscoveredCaptures.ts

```diff
@@ -69,7 +72,10 @@ function useStoreDiscoveredCaptures() {
                 const supabaseConfig: SupabaseConfig | null =
                     editWorkflow && lastPubId
                         ? { catalogName: entityName, lastPubId }
-                        : null;
+                        : {
+                              catalogName: '',
+                              lastPubId: NEW_TASK_PUBLICATION_ID,
+                          };
```

Comment:
Eventually the `catalog name` is optional. What is the value of adding a blank one here?

Reply:
`catalogName` is not an optional property of a `SupabaseConfig`.

Reply:
Yup.